backend/endpoints/rooms.py

```diff
@@ -6,8 +9,10 @@
 from schema.room import Role, Room, RoomMember, RoomRequest, RoomType
 from utils.centrifugo import Events, centrifugo_client
 from utils.db import DataStorage
+from utils.room_utils import ROOM_COLLECTION, remove_mem
```

Comment:
This should be merged with line 13 and rewritten

```suggestion
from utils.room_utils import ROOM_COLLECTION, remove_mem, get_room
```